README.md

```diff
@@ -75,46 +80,46 @@ Running Tests Automatically
 Tests can be run automatically in a browser using the `run` command of
 the `wpt` script in the root of the checkout. This requires the hosts
 file setup documented above, but you must *not* have the
-test server already running when calling `wpt run`. The basic command
+test server already running when calling `wpt run`. The basic command-
```

Comment:
This turns into "command- line", need to put command-line together.

Reply:
ah, good catch - thanks